Sources/Testing/ExitTests/ExitTest.Result.swift

```diff
@@ -0,0 +1,86 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024–2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+@_spi(Experimental)
+#if SWT_NO_EXIT_TESTS
+@available(*, unavailable, message: "Exit tests are not available on this platform.")
+#endif
+extension ExitTest {
+  /// A type representing the result of an exit test after it has exited and
+  /// returned control to the calling test function.
+  ///
+  /// Both ``expect(exitsWith:observing:_:sourceLocation:performing:)`` and
+  /// ``require(exitsWith:observing:_:sourceLocation:performing:)`` return
+  /// instances of this type.
+  public struct Result: Sendable {
+    /// The exit condition the exit test exited with.
+    ///
+    /// When the exit test passes, the value of this property is equal to the
+    /// exit status reported by the process that hosted the exit test.
+    public var statusAtExit: StatusAtExit
```

Comment:
Since this type is (now) `ExitTest.Result`, it feels sort of implied that a status it holds would be the status _at exit_. Like another comment elsewhere, could we omit "AtExit" from the property name?
```suggestion
    public var status: StatusAtExit
```

Reply:
If we renamed the type `ExitStatus`, I'd still want to call it `exitStatus`. `status` is just too generic for my comfort.